feature-libs/customer-ticketing/occ/adapters/occ-customer-ticketing.adapter.spec.ts

```diff
@@ -112,4 +112,35 @@ describe('OccCustomerTicketingAdapter', () => {
       );
     });
   });
+
+  describe('getTickets', () => {
+    it('should get tickets for the given customer id', (done) => {
```

Comment:
Please test when server returns 500 for any reason

Reply:
Since we have a global http interceptor we won't have those tests at this level in the adapter